lib/compat.php

```diff
@@ -9,6 +9,124 @@
  */
 
 if ( ! function_exists( 'register_block_type_from_metadata' ) ) {
+	/**
+	 * Removes the block asset's path prefix if provided.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @param string $asset_handle_or_path Asset handle or prefixed path.
+	 *
+	 * @return string Path without the prefix or the original value.
+	 */
+	function gutenberg_remove_block_asset_path_prefix( $asset_handle_or_path ) {
```

Comment:
Should it be private? Similar to:
https://github.com/WordPress/wordpress-develop/blob/de3fd5b242afd97db2049f74581bc1acd9282d74/src/wp-includes/blocks.php#L410-L420